@types/ical.d.ts

```diff
@@ -0,0 +1,257 @@
+// SPDX-FileCopyrightText: © 2019 EteSync Authors
+// SPDX-License-Identifier: GPL-3.0-only
+// https://github.com/mozilla-comm/ical.js/issues/367#issuecomment-568493517
```

Comment:
Added some decent definitions I've found for this library.

Reply:
This comes with tons of warnings - needs some checking over and at the very least any should become unknown. 🤔